cmd/cosign/cli/options/import_key_pair.go

```diff
@@ -39,4 +41,7 @@ func (o *ImportKeyPairOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&o.OutputKeyPrefix, "output-key-prefix", "o", "import-cosign",
 		"name used for outputted key pairs")
 	_ = cmd.Flags().SetAnnotation("output-key-prefix", cobra.BashCompFilenameExt, []string{})
+
+	cmd.Flags().BoolVarP(&o.SkipConfirmation, "yes", "y", false,
+		"skip confirmation prompts for non-destructive operations")
```

Comment:
Question: is file overwrite considered as destructive operation? Should we change it to a different name?

Reply:
Yes, this should say destructive. Given the simplicity of this command, you could just say "skip confirmation prompts for overwriting existing key"